Composer/packages/tools/language-servers/language-generation/src/LGServer.ts

```diff
@@ -117,7 +117,7 @@ export class LGServer {
 
     this.connection.onRequest((method, params) => {
       if (initializeDocumentsMethodName === method) {
-        const { uri, lgOption }: { uri: string; lgOption?: LGOption } = params;
+        const { uri, lgOption }: { uri: string; lgOption?: LGOption } = params as { uri: string; lgOption?: LGOption };
```

Comment:
Maybe extract `{ uri: string; lgOption?: LGOption }` out to a type since it's used twice. Or not.